src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -258,17 +257,10 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
 
   const backButtonOptionList = useMemo(
     () =>
-      ownedVenues?.filter(({ id, name, template, worldId }) => {
-        if (venueId === id || worldId !== roomVenue?.worldId) {
-          return null;
-        }
-
-        return {
-          name,
-          template,
-        };
-      }) ?? ALWAYS_EMPTY_ARRAY,
-    [ownedVenues, roomVenue?.worldId, venueId]
+      ownedVenues?.filter(
```

Comment:
When one looks at this, one might think... "hm... this one could be undefined also", but it is always defined